File: MusicPlayer/app/src/main/java/com/example/musicplayer/MainActivity.java
package com.example.musicplayer;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    TextView next1;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        id();
    }

    private void id()
    {
        next1=findViewById(R.id.next1);

        next1.setOnClickListener( v -> {

            Intent i = new Intent(this,AdSecondActivity.class);
            startActivity(i);
            finish();
        });
    }
}

File: Touradvisor/app/src/main/java/com/example/touradvisor/Activity/MainActivity.kt
package com.example.touradvisor.Activity

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import androidx.appcompat.app.AppCompatActivity
import com.example.touradvisor.R

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


        Handler().postDelayed({
            val intent = Intent(this@MainActivity, WelcomeScreenActivity::class.java)
            startActivity(intent)
            finish()
        }, 1000)

    }
}